.rultor.yml

```diff
@@ -1,14 +1,12 @@
-install:
```

Comment:
see https://github.com/coala/coala/issues/4876

Reply:
Whoops, accidentally removed that one